.pre-commit-config.yaml

```
@@ -37,7 +37,7 @@ repos:
     - id: check-yaml
   # Leave black at the bottom so all touchups are done before it is run.
   - repo: https://github.com/ambv/black
```

Since you are updating the version, could you also update the repo please?
```suggestion
  - repo: https://github.com/psf/black
```